src/main/java/org/opentripplanner/apis/gtfs/mapping/RouteRequestMapper.java

```diff
@@ -57,6 +57,7 @@ public static RouteRequest toRouteRequest(
       context.transitService().getTimeZone()
     );
 
+    callWith.argument("wheelchair", request::setWheelchair);
```

Comment:
This is a duplicate line - you can remove it.

Reply:
Refactored in https://github.com/ibi-group/OpenTripPlanner/pull/197/commits/64d521f8fa59570666fcd53eec629db5a5699fd4.